### DISCUSSION TITLE
The position of sensor relative to vehicle local coordinate system

### DISCUSSION THREAD

**Discussion by RO-Caiyun** [microsoft/AirSim#3461] [Support Q&A]
Hi guys!
     I want to know the origin position of IMU and GPS sensors，but i didn't find it in docs.Did anyone know how to get the information?
Thanks in advance!

**Comment by rajat2004** [comment]
AFAIK, both IMU and GPS sensors are positioned at the centre of the vehicle, and don't have any relative positioning, will need to dig through the code and some old issues though for more info

**Reply by RO-Caiyun** [reply]
Thank you!You've been most helpful!
